beacon-chain/blockchain/process_block.go

```diff
@@ -558,6 +559,14 @@ func (s *Service) isDataAvailable(ctx context.Context, root [32]byte, signed int
 	// The gossip handler for blobs writes the index of each verified blob referencing the given
 	// root to the channel returned by blobNotifiers.forRoot.
 	nc := s.blobNotifiers.forRoot(root)
+	nextSlot := slots.BeginsAt(signed.Block().Slot()+1, s.genesisTime)
+	// Avoid underflow if Now is somehow after the next slot start.
+	nextSlotDur := time.Duration(0)
+	if nextSlot.After(time.Now()) {
+		nextSlotDur = time.Until(nextSlot)
+	}
+	slotEnd := time.NewTimer(nextSlotDur)
```

Comment:
I think this may cause a timer that expires very quickly still, the guarantee that `time.NewTimer()` gives is that it has a *minimum* wait time. 

I'd rather simply not log this at all if the block is coming after the slot's end.